### PR TITLE
Update syllabus.yml

### PR DESCRIPTION
**Pull request by danielzgsilva**

### CODE REVIEW

**Review by SirRoboto — COMMENTED** [review]
are we still changing gpu enable to true from false @ionlights?

File: sp20/syllabus.yml [hunk 1]
```diff
@@ -79,7 +83,7 @@ meeting03:
       datasets: []
       competitions: []
```
Comment (brandons209) [inline sp20/syllabus.yml:84]: add comp following the naming scheme: "ucfai-core-sp20-cnns"

**Review by brandons209 — CHANGES_REQUESTED** [review]

File: sp20/syllabus.yml (hunk 1 above)
Comment (jmuchovej) [reply sp20/syllabus.yml:84]: don't add the competition. `autobot` does that already.

**Review by jmuchovej — DISMISSED** [review]

File: sp20/syllabus.yml [hunk 2]
```diff
@@ -77,9 +81,9 @@ meeting03:
       youtube: ''
     kaggle:
       datasets: []
-      competitions: []
+      competitions: ["ucfai-core-sp20-cnns"]
```
Comment (jmuchovej) [inline sp20/syllabus.yml]: @brandons209 said you need it – you actually don't. 😅 that's my b on explaining.

**Review by brandons209 — APPROVED** [review]

**Pull request merged**